tests/cpp/test_gpu3.cpp

```diff
@@ -8943,11 +8943,14 @@ TEST_F(NVFuserTest, ReplaceSymbolicSizesPreferSimplerExtents) {
   fusion.addInput(tv0);
   auto tv1 = makeSymbolicTensor(2);
   fusion.addInput(tv1);
-  auto i0 = IrBuilder::create<Val>(DataType::Index);
-  fusion.addInput(i0);
 
-  auto tv2 = reshape(tv0, {i0});
-  auto tv3 = reshape(tv1, {i0});
+  auto tv2 = reshape(
```

Comment:
This happened to work due to the bug in replaceSymbolicSizes, but it should have been written this way from the beginning.